docs/platform/howto/btar/enable-backup-to-another-region.md

```diff
@@ -14,7 +15,8 @@ To add an additional service backup for your service, you can use the Aiven
 
 ## Prerequisites
 
-- Enable BTAR for your organization by [contacting the sales team](mailto:sales@aiven.io).
+- This feature is in [limited availability](/docs/platform/concepts/beta_services).
+  Enable it by contacting your account team.
```

Comment:
nit: may be keep this line consistent across all topics? The about topic has this: 
Contact your account team to enable it.

docs/platform/concepts/backup-to-another-region.md

Reply:
> nit: may be keep this line consistent across all topics? The about topic has this: Contact your account team to enable it.
> 
> docs/platform/concepts/backup-to-another-region.md

Actually, this reverted syntax here is for purpose. In the prerequisites, I wanted to emphasize the task/action the user needs to take as a prerequisite hence putting "enable" at the beginning of the sentence.